kolibri/plugins/style_guide/assets/src/views/shell/sidenav.vue

```diff
@@ -35,12 +35,14 @@
 
 <style lang="stylus" scoped>
 
+  $left-padding = 25px
```

Comment:
Since this is used in multiple files, we should probably define them in a centralized global style file and refer to it here?